spec/serializers/personal_information_serializer_spec.rb

```diff
@@ -1,20 +1,37 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require 'va_profile/demographics/demographic_response'
 
 RSpec.describe PersonalInformationSerializer do
-  subject { serialize(mvi_profile, serializer_class: described_class) }
-
-  let(:mvi_profile) { MPIData.for_user(create(:user, :loa3)).profile }
-  let(:attributes) { JSON.parse(subject)['data']['attributes'] }
+  before do
```

Comment:
Is there a reason why let is not being used to define variables?

Reply:
```
  let(:demographics) { get_demographics }
  let(:response) { serialize(demographics, serializer_class: described_class) }
  let(:attributes) { JSON.parse(response)['data']['attributes'] }
```

Reply:
HI @RachalCassity - no, I can make that change.

Reply:
Thanks @RachalCassity!